docs/markdown/connecting-to-testnet.md

```diff
@@ -0,0 +1,58 @@
+# Connecting to an ILP Testnet
+
+This tutorial will help you start an Interledger node locally on your computer and connect to an external ILP testnet.
```

Comment:
> connect to an external ILP testnet.

connect to an ILP testnet node.

Reply:
https://github.com/interledger/interledger.github.io/pull/108/commits/040b4c13cc77219844de0e840d34a66e26207ce1#diff-95b449acc2e692458b91372a5b6e33e7R3